### PR TITLE
Make HTML website-compatible

### PR DESCRIPTION
**Pull request by RoiEXLab**
I'm working on an travis script for the website repo which requires those changes.
Java should be ok with that as CSS like this is already being used in some descriptions

### CODE REVIEW

File: triplea_maps.yaml [hunk 1]
```diff
@@ -926,7 +926,7 @@
   description: |
     <div style="text-align: left;">
     	<br>
-    	<h1 style="text-align: left; font-size:120%; color: #000000;">#NEW WORLD ORDER#</h1>
+    	<h2 style="text-align: left; font-size:120%; color: #000000;">#NEW WORLD ORDER#</h2>
```
Comment (RoiEXLab) [inline triplea_maps.yaml]: h1 tags need to be unique on a page...

File: triplea_maps.yaml [hunk 2]
```diff
@@ -88,10 +88,10 @@
   version: 1
   img: http://tripleamaps.sourceforge.net/images/TripleA_total_world_war_mini.png
   description: |
-    <br><em><b><font size="6">Created by Rolf Larsson and Hepster</font></b></em>
+    <br><em><b style="font-size: 6px;">Created by Rolf Larsson and Hepster</b></em>
```
Comment (ssoloff) [inline triplea_maps.yaml]: I'm not sure HTML `<font size="6">` is equivalent to CSS `font-size: 6px`.  According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font), the `size` attribute seems to be a browser-specific predefined size, not necessarily pixels.  Take a look at [this](http://jsbin.com/cenumiyume/1/edit?html,output) JSBin that demonstrates the difference.

Comment (RoiEXLab) [inline triplea_maps.yaml]: I see...

Why is such a system even invented?

Comment (ssoloff) [inline triplea_maps.yaml]: :smile: I'd recommend changing your `game_engine.properties` to point to your locally-modified `triplea_maps.yaml` so you can see how the Download Maps window renders these map descriptions.  That way you can find the new CSS `font-size` (in whatever unit) that is equivalent to the old HTML `font` `size` for the Swing HTML engine.

Comment (ssoloff) [inline triplea_maps.yaml]: This [post](https://stackoverflow.com/questions/20805668/what-is-best-replacement-for-font-size-x-in-css) shows the equivalents in terms of named sizes.  I confirmed that CSS size `xx-large` does indeed render the same as HTML size `6`.

Comment (ssoloff) [inline triplea_maps.yaml]: I should have clarified in my previous comment that the equivalence between `xx-large` and `6` was on Firefox.  As this is a browser-specific mapping, the Swing renderer can use a different mapping, and it apparently does by mapping `6` to `x-large` as shown below:

font-size

**Pull request merged**